estate/views/estate_property_type_views.xml

```diff
@@ -13,11 +13,16 @@
       <field name="arch" type="xml">
         <form string="Property Type">
           <sheet>
-            <group>
+            <div class="oe_title">
                 <h1>
-                    <field name="name"/>
+                    <field name="name" />
                 </h1>
-            </group>
+            </div>
+
+            <h1>
+                <field name="name"  />
+            </h1>
+
```

Comment:
There are two field names here, which one is correct? 

choose one and remove the other please